src/app/components/account/account-list-item.layout.tsx

```diff
@@ -40,12 +40,7 @@ export function AccountListItemLayout(props: AccountListItemLayoutProps) {
         titleLeft={accountName}
         titleRight={
           isLoading ? (
-            <Spinner
-              color="accent.text-subdued"
-              position="absolute"
-              right={0}
-              top="calc(50% - 8px)"
-            />
+            <Spinner color="ink.text-subdued" position="absolute" right={0} top="calc(50% - 8px)" />
```

Comment:
I missed it on your PR, so sorry but maybe we don't need the `ink` nesting anymore? 

We had: 
````
semanticTokens = {
  colors: {
    // Primitive colors defined as semantic tokens to match Radix
    ink: {
         accent: {
           'text-primary': ....
```

If we are getting rid of the scale and ink naming, maybe we can flatten this down to be: 
```
semanticTokens = {
  colors: {
       'text-primary': 
 ....

```

Then we can avoid having `ink.text-subdued` and instead have just `text-subdued`? 

We can do that in the next iteration

Reply:
I think we should keep the ink as it helps differentiate better, and serves a semantic purpose of identifying the code foreground/background colour. Pretty sure this is how they do in Figma too @fabric-8 ?

Reply:
Yeah, I was following the design team here and the specs provided.